### PR TITLE
Add timezone conversion.

### PR DESCRIPTION
**Pull request by Shylock-Hg**
Require https://github.com/vesoft-inc/nebula-common/pull/571

### CODE REVIEW

File: src/client/Session.cpp [hunk 1]
```diff
@@ -48,4 +50,17 @@ void Session::release() {
     }
 }
 
+void Session::toLocal(DataSet &data, int32_t offsetSecs) {
```
Comment (laura-ding) [inline src/client/Session.cpp:53]: You don't have to provide a method like that, but you can tell user how to get localTime and get localDateTime in example code

Comment (Shylock-Hg) [reply src/client/Session.cpp:53]: I will add example.

Comment (laura-ding) [reply src/client/Session.cpp:53]: The interface better move to test code, no need to provide it to the user

Comment (Shylock-Hg) [reply src/client/Session.cpp:53]: User could use or not.

**Review by laura-ding — DISMISSED** [review]

**Comment by laura-ding** [comment]
Please fix the ci test

**Review by laura-ding — APPROVED** [review]

**Pull request merged**